modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/processor/AbstractPipelineProcessor.java

```diff
@@ -210,12 +212,14 @@ protected PipelineWorkload pollTask(long timeout, TimeUnit unit) throws Interrup
      * @param taskCls Task class.
      * @param workload Argument.
      * @param lsnr Listener.
+     * @throws InterruptedException If task was interrupted.
+     * @throws IgniteException If failed to execute the task.
      */
     protected <T extends CachePartitionRequest, R> void compute(
         Class<? extends ComputeTask<T, ExecutionResult<R>>> taskCls,
         T workload,
         IgniteInClosure<? super R> lsnr
-    ) throws InterruptedException {
+    ) throws InterruptedException, IgniteException {
```

Comment:
IgniteException is a RuntimeException right?

Reply:
That is right.